R.I.P Hassbian (#10933)
home-assistant/home-assistant.github.io: source/_posts/2019-10-25-rip-hassbian.markdown

---
layout: post
title: "R.I.P Hassbian"
description: "The end of Hassbian"

> date: 2019-10-25 01:11:03

Reviewer: We should make sure this date is in the past when we hit publish, or else it won't be published.

Author: Current date should be good as it is but time might need to be changed depending on the timezone of the servers.

Reviewer: Time is UTC.

date_formatted: "October 25, 2019"
author: Fredrik Lindqvist
author_twitter: Landrash
comments: true
categories: community
---

> It's time for the part of Home Assistant that once was the simplest way to install Home Assistant to retire. Hassbian is being retired.

Reviewer 2: > It's time for the part of Home Assistant that once was the simplest way to install Home Assistant to retire. Hassbian is being retired.

Feels clunky. Maybe:

_It's time for what was once the simplest way to install Home Assistant to retire._

Author: Love it. Thank you :)

## R.I.P Hassbian

> Some off you will probably wonder why Hassbian is being retired and I'll try to give a proper motivation. 

Reviewer 3: ```suggestion
Some of you will probably wonder why Hassbian is being retired and I'll try to give a proper motivation. 
```

> First of is the age old factor of **time**. I as the lead developer of Hassbian hasn't been able to give it the time and attention it requires and there has been few others that have pushed the project further. The one exception being [@ludeeus] who's done most of the heavy lifting when the image was retooled to use a proper apt package and repository.

Reviewer 2: I as the lead developer of Hassbian haven't been able to give it the time and attention it requires, and there have been few others that have pushed the project further.

> First of is the age old factor of **time**. I as the lead developer of Hassbian hasn't been able to give it the time and attention it requires and there has been few others that have pushed the project further. The one exception being [@ludeeus] who's done most of the heavy lifting when the image was retooled to use a proper apt package and repository.

Reviewer 3: ```suggestion
First of is the age-old factor of **time**. I as the lead developer of Hassbian hasn't been able to give it the time and attention it requires, and there have been few others that have pushed the project further. The one exception being [@ludeeus] who's done most of the heavy lifting when the image was retooled to use a proper apt package and repository.
```

Second is that it's no longer the best option for most to use as an installation method. Hass.io has surpassed Hassbian in almost all ways I can imagine.

## Next Step for Hassbian

> Since Hassbian has been around for quite a while there's quite a few users that don't want Hassbian to go away. To make this as easy as possible for those users here's the plan.

Reviewer 3: ```suggestion
Since Hassbian has been around for quite a while, there are quite a few users that don't want Hassbian to go away. To make this as easy as possible for those users, here's the plan.
```

>  - The repositores [pi-gen] and [hassbian-scripts] hosting the Hassbian projects files will be moved to a new organization. 

Reviewer 2: repositories

>  - The repositores [pi-gen] and [hassbian-scripts] hosting the Hassbian projects files will be moved to a new organization. 

Reviewer 3: ```suggestion
 - The repositories [pi-gen] and [hassbian-scripts] hosting the Hassbian project files will be moved to a new organization. 
```

>  - A last release will be released by me mid Q4 2019. Image hosted under the [pi-gen] repository. 

Reviewer 2: A last release will be done by me mid Q4 2019. This image will be hosted under the [pi-gen] repository.

>  - The [pi-gen] repository will be reworked to work with a standard [raspbian] image with minor modifications for anyone wanting to create their own "Hassbian like" image. This has always been possible but the current repository is a bit out of date with the current layout of the Raspbian image. 

Reviewer 3: ```suggestion
 - The [pi-gen] repository will be reworked to work with a standard [raspbian] image with minor modifications for anyone wanting to create their own "Hassbian like" image. This has always been possible, but the current repository is a bit out of date with the current layout of the Raspbian image. 
```

>  - The [hassbian-scripts] package will get a final release and will continue to be hosted on Gitlab. Some minor changes will be made to reflect the changes to the Project.

Reviewer 3: ```suggestion
 - The [hassbian-scripts] package will get a final release and will continue to be hosted on Gitlab. Some minor changes will be made to reflect the changes to the project.
```

## Next step for Hassbian users
The Hassbian image has always aimed to be the same as a manual Raspbian Lite installation with some packages added.

> There won't really be any big changes for all of the users of Hassbian and for documentation please refer to the [Manual installation on a Raspberry PI] method.

Reviewer 3: ```suggestion
There won't really be any big changes for all of the users of Hassbian and for documentation, please refer to the [Manual installation on a Raspberry PI] method.
```

## Last but not least

> Last but not least thank you to all of those who contributed in anyway to the Hassbian Project and image.

Reviewer 3: ```suggestion
Last but not least, thank you to all of those who contributed, in any way, to the Hassbian project and image.
```

 - The repositories [pi-gen] and [hassbian-scripts] hosting the Hassbian projects files will be moved to a new organization. 
 - A last release will be done by me mid Q4 2019. This image will be hosted under the [pi-gen] repository. 
 - The [pi-gen] repository will be reworked to work with a standard [raspbian] image with minor modifications for anyone wanting to create their own "Hassbian like" image. This has always been possible, but the current repository is a bit out of date with the current layout of the Raspbian image. 
 - The [hassbian-scripts] package will get a final release and will continue to be hosted on Gitlab. Some minor changes will be made to reflect the changes to the project.

## Next step for Hassbian users
The Hassbian image has always aimed to be the same as a manual Raspbian Lite installation with some packages added.
There won't really be any big changes for all of the users of Hassbian and for documentation, please refer to the [Manual installation on a Raspberry PI] method.

## Alternatives

> If your want to continue using something similar have a look at the [manual installation on a Raspberry Pi] since it is the base Hassbian was created from.

Reviewer 3: ```suggestion
If you want to continue using something similar, have a look at the [manual installation on a Raspberry Pi] since it is the base Hassbian was created from.
```

> For everyone else I would wholeheartedly recommend Hass.io since it what I personally use now (It's what I had hoped Hassbian could have been but better).

Reviewer 3: ```suggestion
For everyone else, I would wholeheartedly recommend Hass.io since it what I personally use now (It's what I had hoped Hassbian could have been but better).
```

The Hassbian image has always aimed to be the same as a manual Raspbian Lite installation with some packages added.
There won't really be any big changes for all of the users of Hassbian and for documentation, please refer to the [Manual installation on a Raspberry PI] method.

## Alternatives

> If your want to continue using something similar have a look at the [manual installation on a Raspberry Pi] since it is the base Hassbian was created from.

Reviewer 2: If you want to continue using something similar have a look at the [manual installation on a Raspberry Pi] since it is the base Hassbian was created from.

Reviewer 3: Well spotted, added it as a commit suggestion below.
